internal/state/state_test.go

```diff
@@ -221,35 +216,34 @@ func TestABCIResponsesSaveLoad2(t *testing.T) {
 func TestValidatorSimpleSaveLoad(t *testing.T) {
 	tearDown, stateDB, state := setupTestCase(t)
 	defer tearDown(t)
-	assert := assert.New(t)
 
 	statestore := sm.NewStore(stateDB)
 
 	// Can't load anything for height 0.
 	_, err := statestore.LoadValidators(0)
-	assert.IsType(sm.ErrNoValSetForHeight{}, err, "expected err at height 0")
+	assert.IsType(t, sm.ErrNoValSetForHeight{}, err, "expected err at height 0")
 
 	// Should be able to load for height 1.
 	v, err := statestore.LoadValidators(1)
-	assert.Nil(err, "expected no err at height 1")
-	assert.Equal(v.Hash(), state.Validators.Hash(), "expected validator hashes to match")
+	require.NoError(t, err, "expected no err at height 1")
```

Comment:
Any reason that these are changed from `assert` -> `require`? In general, I think we can make this `assert` -> `require` update everywhere that we're checking for an error to be `nil`, since it's unlikely that the test is going to behave correctly once it's encountered an error we did not expect it to.

Reply:
I agree with your logic. I tried to make this change in as many places as I could easily spot.